plugins/Toolbox/resources/qml/ToolboxDownloadsGridTile.qml

```diff
@@ -34,10 +34,11 @@ Item
             Image
             {
                 anchors.centerIn: parent
-                width: UM.Theme.getSize("toolbox_thumbnail_small").width - 26
-                height: UM.Theme.getSize("toolbox_thumbnail_small").height - 26
+                width: UM.Theme.getSize("toolbox_thumbnail_small").width - UM.Theme.getSize("default_margin").width
```

Comment:
Good catch. Only thing is that default margin is 13px so this should be minus `"wide_margin"`

Reply:
Fixed in 1f675f5